recipe/meta.yaml

```diff
@@ -1,39 +1,44 @@
-{% set version = "1.3.2" %}
+{% set name = "blosc" %}
+{% set version = "1.4.4" %}
+{% set sha256 = "76ae6e1cecddb320b5567ba78ee9a2ee31895ecba838fdf05fdb9131939a4705" %}
 
 package:
-  name: python-blosc
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/Blosc/python-blosc/archive/v{{ version }}.tar.gz
-  sha256: 63d1cd1da14087fa69bffc513b25d59deb24f45c65e4406acafaaa6ec9bd7cf2
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
   number: 0
-  script: python setup.py install --blosc=$PREFIX --single-version-externally-managed --record record.txt
-  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - blosc
   run:
     - python
-    - blosc
```

Comment:
I guess the question will arise how we build `c-blosc`/`blosc` on Windows.

@msarahan can you please advise us on this? 😄